public/components/event_analytics/explorer/visualizations/config_panel/config_panes/config_controls/config_switch.tsx

```diff
@@ -14,7 +14,7 @@ interface EUISwitch {
 }
 export const ConfigSwitch: React.FC<EUISwitch> = ({ label, disabled, checked, handleChange }) => (
   <Fragment key={`config-switch-${label}`}>
-    <EuiFormRow label={label}>
+    <EuiCompressedFormRow label={label}>
       <EuiSwitch
```

Comment:
Should this be changed to a compressed switch?

Reply:
Nah; it already has the compressed attrib.